src/client/containers/QuizPage/QuizPage.container.js

```diff
@@ -81,60 +88,79 @@ export const QuizPage = () => {
-              imageSrc={QuizQuestions[currentOn].image}
-              question={QuizQuestions[currentOn].question}
-            />
-            <div>
-              <QuizAnswers
-                isAgreementQuestion={
-                  QuizQuestions[currentOn].isAgreementQuestion
-                }
-                firstAnswer={QuizQuestions[currentOn].firstAnswer}
-                secondAnswer={QuizQuestions[currentOn].secondAnswer}
-              />
-            </div>
+            {isLoading && <div>Loading...</div>}
+            {error && <div>{error.message}</div>}
+            {questions && (
+              <>
+                <QuestionComponent
+                  imageSrc={currentQuestion.image_url}
+                  question={currentQuestion.question}
+                />
+                <div>
+                  <QuizAnswers
+                    selectedAnswer={selectedAnswer}
+                    setSelectedAnswer={setSelectedAnswer}
+                    isAgreementQuestion={!currentQuestion.is_agreement_question}
+                    answers={currentQuestion.answers}
+                  />
+                </div>
+              </>
+            )}
           </div>
         </div>
-        <div className="button-page">
-          {currentOn !== 0 && currentOn !== QuizQuestions.length - 1 && (
-            <span className="back-button">
-              <Buttons
-                label="Back"
-                color="grey"
-                size="big"
-                isMono={false}
-                onClick={goToPrevious}
-              />
-            </span>
-          )}
-          {currentOn !== QuizQuestions.length - 1 ? (
-            <span className="next-button">
-              <Buttons
-                label="Next"
-                size="big"
-                isMono={false}
-                onClick={handleQuestions}
-              />
-            </span>
-          ) : (
-            <span className="results">
-              <Buttons
-                label="See My Results"
-                size="big"
-                isMono={false}
-                onClick={handleClick}
+        {questions && (
+          <>
+            <div className="button-page">
+              {currentOn !== 0 && currentOn !== questions.length - 1 && (
+                <span className="back-button">
+                  <Buttons
+                    label="Back"
+                    color="grey"
+                    size="big"
+                    isMono={false}
+                    onClick={goToPrevious}
+                  />
+                </span>
+              )}
+              {currentOn !== questions.length - 1 ? (
+                <span className="next-button">
+                  <Buttons
+                    label="Next"
+                    size="big"
+                    isMono={false}
+                    onClick={handleQuestions}
+                  />
+                </span>
+              ) : (
+                <span className="results">
+<Link
```

Comment:
Please look into using prettier so the code is better organized

Reply:
I do use Prettier. Reason for this is the same as previously answered.